go.mod

```diff
@@ -13,7 +16,7 @@ require (
 	github.com/containerd/containerd v1.6.8
 	github.com/daviddengcn/go-colortext v1.0.0
 	github.com/docker/cli v20.10.17+incompatible
```

Comment:
Please replace the version here as well.

Reply:
Done.